src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -362,25 +371,28 @@ describe('Hides fields based on user permissions', () => {
 
   it('hides opting out and zendesk tickets for non edX staff', async () => {
     mockAuthentication(false);
-    await act(async () => render(<ProctoredExamSettings {...defaultProps} />));
+    await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
     expect(screen.queryByTestId('allowOptingOutYes')).toBeNull();
     expect(screen.queryByTestId('createZendeskTicketsYes')).toBeNull();
   });
 
   it('shows opting out and zendesk tickets for edX staff', async () => {
     mockAuthentication(true);
-    await act(async () => render(<ProctoredExamSettings {...defaultProps} />));
+    await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
     expect(screen.queryByTestId('allowOptingOutYes')).not.toBeNull();
     expect(screen.queryByTestId('createZendeskTicketsYes')).not.toBeNull();
   });
 });
 
 describe('ProctoredExamSettings connection states tests', () => {
   it('shows the spinner before the connection is complete', async () => {
```

Comment:
@schenedx this is the test that was causing issues.  It's not clear to me exactly how this was working before since we never mocked any of the backend calls yet the spinner was successfully removed?

Reply:
So, this test was working before because I didn't wrap the `render` call in the `await act()` call. This means, the test will try to render and immediately execute the `screen.getByTestId` function. That is what we want for test for spinner before the AJAX call completes. 
Your change here made the API call not complete because you are returning an extra promise. This is acceptable but definitely not ideal. You can merge this PR and I can tinker with this when I have time.